### ISSUE TITLE
Umlaute in apa_table

### ISSUE THREAD

**Issue by TobiasKoeln** [crsh/papaja#133]
Using Umlaute or é, è in the captions or notes in the apa_table function does not work (at my computer). This could be useful if creating a table like this one:

````{r, results="asis"}
my_table <- apply(cars, 2, function(x)
   c(Mean = mean(x), SD = sd(x), Min = min(x), Max = max(x))
)

#This one works
apa_table(my_table
          , caption = "This is a uber nice table"
          , note = "Papaja is trs nice")

#This one doesn't
apa_table(my_table
          , caption = "This is a über nice table"
          , note = "Papaja is très nice")
````

Is it just me (again) or could this feature be built into the function?

papaja 0.1.0.9492 (master)
Rstudio 1.0.143
R 3.4.1

**Comment by crsh** [comment]
Hi Tobi,

it's not just you. I don't fully understand why this happens. I saved my document using UTF-8 encoding assuming that `\usepackage[utf8]{inputenc}`, as is included in the template, should enable support for special characters. As you point out it doesn't. So I need to look into this more closely.

I did, however, get it to work by using a different latex engine. Try using either XeLaTeX or LuaLaTeX and let me know if that works.

```
output            :
  papaja::apa6_pdf:
    latex_engine: xelatex
```

**Comment by crsh** [comment]
I've been exploring this a little more and although I haven't found a real fix for this there is another work around:

```{r}
apa_table(my_table
          , caption = "This is a \\''uber nice table"
          , note = "Papaja is tr\\`es nice")
          , escape = FALSE)
```

This is writing special characters the LaTeX way without relying on unicode characters.

I have found this to be an issue of the `papaja` TeX template. The same problem occurs with `knitr::kable()` but it works with either function in regular `rmarkdown::pdf_document()` or `bookdown::pdf_document2()`-files.

**Comment by crsh** [comment]
Okay, wow. That was an intense hunt. It turns out this is a [bug](https://tex.stackexchange.com/questions/144372/error-when-using-endfloat-with-unicode-characters/144425) in the LaTeX package `endfloat` (fixed as of version 2.6, if I [understand](https://github.com/axelsommerfeldt/endfloat/blob/master/README#L58) correctly). 😱

Could you try adding the following to your TeX preamble (e.g. by editing the TeX file that is produced but not rendered), to see if that fixes the problem?

~~~tex
\makeatletter
\renewcommand\efloat@iwrite[1]{\expandafter\immediate\expandafter\protected@write\csname efloat@post#1\endcsname{}}
\makeatother
~~~

**Comment by TobiasKoeln** [comment]
Works! Thank you
